Android/backend/README.md

```diff
@@ -0,0 +1,10 @@
+# Intra Backend
+
+Intra backend is a Go project using Outline SDK.
+
+## Build
+
+```sh
+gomobile clean
+gomobile bind -target=android -o ./Android/backend/backend.aar ./Android/backend/intra
```

Comment:
I see the api is all in one package. Nice.

Can you move it so that there aren't things nested in it?
I was imagining something like

- go/
  - backend/ (or some name that is clearer)
  - internal/
    - doh/
    - sni/
    - vpn/  (we should move the vpn stuff out of doh)